pkg/backend/apply.go

```diff
@@ -125,6 +125,16 @@ func PreviewThenPrompt(ctx context.Context, kind apitype.UpdateKind, stack Stack
 		return plan, changes, nil
 	}
 
+	if kind == apitype.UpdateUpdate &&
```

Comment:
Should probably be for previews as well

Reply:
I've looked through the codebase and `preview` actually invokes `apply` directly and doesn't go through this pathway as it returns a `*deploy.Plan` the changes to include this on preview, would be much larger and would change a lot of function signatures.

Reply:
Ugh backend code.
We should fix this, a vast amount of the value here will be pointing this out to people at preview time.